webview-ui/src/utils/transformation.ts

```diff
@@ -17,19 +17,3 @@ export const sanitizeIncidents = (incidents: Incident[]): Incident[] =>
       // expect 1-based numbering (vscode.Position is zero-based)
       lineNumber: Number.isInteger(it.lineNumber) && it.lineNumber! > 0 ? it.lineNumber : 1,
     }));
-
-export const groupIncidentsByMsg = (
```

Comment:
This was a temp placeholder until we will be able to share code via `shared` project. This was fixed in https://github.com/konveyor/editor-extensions/pull/193 . 
Now we should be able to use this functionality from https://github.com/konveyor/editor-extensions/blob/2f4a371688f41199a09318adc22baa9f7cd7c1f9/shared/src/transformation.ts#L21
This is important to keep the groupings in Issue tree view (and possibly other components) look exactly the same as the groupings used in the webview.

Reply:
Ah, I see what Radek is saying.

Incidents can be grouped by description/message under their Issue.  The issue tree view shows per description/message while the analysis view is back to per Issue.  We can solve this in future.  I'll kick it to an issue so we can have a discussion about it and display things the same way in both views.

Reply:
+1

Reply:
https://github.com/konveyor/editor-extensions/issues/325

Reply:
I'll just put some more details in that issue you opened!  :+1: